js/main.js

```diff
@@ -329,6 +329,25 @@ define([
             }));
           }
           registry.byId("mainWindow").layout();
+          
+          // *******************************************
+          // **** Maptiks Changes below
+          // *******************************************
+          
+          require(['maptiks'], function (mapWrapper) {
+            var container = response.map.container;
+            var maptiksMapOptions = {
+                extent: response.map.extent,
+                maptiks_trackcode: "c311cf16-ad79-42b1-97f9-f433be6c8b00",
+                maptiks_id: "basictest"
```

Comment:
you need to update the configPanel here (https://github.com/sparkgeo/storymap-basic/blob/master/resources/configurationPanel.js). This is similar to how we did the old templates.